.CI/build_all

```diff
@@ -14,11 +14,16 @@ if [ -z "$GH_TOKEN" ]; then
     # Install and configure conda environment.
     curl -L -O https://raw.githubusercontent.com/pelson/Obvious-CI/master/bootstrap-obvious-ci-and-miniconda.py
     python bootstrap-obvious-ci-and-miniconda.py ~/miniconda x64 3 --without-obvci && source ~/miniconda/bin/activate root
+    mkdir -p ~/miniconda/conda-meta
+    echo 'conda-build 1.21.7' > ~/miniconda/conda-meta/pinned
```

Comment:
I would go for a more explicit approach of pining conda-build right below the installation of `conda-forge-build-setup`. (I know that `conda-forge-build-setup` made this process more difficult b/c it hides what is being installed...)